test/e2e/page-objects/pages/account-list-page.ts

```diff
@@ -377,6 +385,15 @@ class AccountListPage {
     await this.driver.waitForSelector(this.addEthereumAccountButton);
   }
 
+  async openConnectHardwareWalletModal(): Promise<void> {
```

Comment:
Hey Chloe,

Could we add comments to the methods in this file to make the code easier to follow?

Also, would it make sense for this method to return the `ConnectHardwareWallet` page directly? It seems like that's what every caller ends up creating right after. Curious to hear your thoughts!